draft-ietf-capport-architecture.xml

```diff
@@ -112,24 +112,40 @@
        principles:
```

Comment:
Do we still want to call these principles?

Reply:
They seem like requirements, with explanatory notes.